README.md

```diff
@@ -236,13 +228,7 @@ However, you may opt for them by enabling `spof` feature, then you can use `trus
 for key import and `key_share::reconstruct_secret_key` for key export.
 
 ## Differences between the implementation and CGGMP24
-[CGGMP24] only defines a non-threshold protocol. To support general thresholds,
-we defined our own CGGMP24-like key generation and threshold signing
-protocols. However, we keep both
-threshold and non-threshold versions of the protocols in the crate, so if you opt for the non-threshold
-protocol, you will be running the original protocol defined in the paper.
-
-There are other (small) differences in the implementation compared to the original paper (mostly typo fixes);
+There are small differences in the implementation compared to the original paper [CGGMP24] (mostly typo fixes);
```

Comment:
This is still true as well:

- cggmp only defines a non-threshold protocol (and briefly says that adding thresholdness is possible)
- library is structured in the way that user can opt out thresholdness, so original protocol is carried out